<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SCIL-GWL-295" failures="2" tests="11" name="Default test" time="168.716" errors="0" timestamp="2025-10-03T13:06:46 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.179"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="22.816"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.689"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.379"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="11.023"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.072"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.221"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.019"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.893">
    <failure type="java.lang.AssertionError" message="Submit button is still enabled expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Submit button is still enabled expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at stepDefinitions.RecipeSteps.submit_button_should_be_disabled(RecipeSteps.java:127)
at ✽.Submit button should be disabled(file:///D:/eclipse-jee-2025-06-R-win32-x86_64/RecipeAutomation3/src/test/java/features/recipe.feature:55)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.778">
    <failure type="java.lang.AssertionError" message="Recipe name exceeded max length of 30! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Recipe name exceeded max length of 30! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.RecipeSteps.recipe_name_field_should_accept_the_value(RecipeSteps.java:143)
at ✽.Recipe name field should accept the value(file:///D:/eclipse-jee-2025-06-R-win32-x86_64/RecipeAutomation3/src/test/java/features/recipe.feature:61)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.669"/>
</testsuite> <!-- Default test -->
